lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
+      else
+        modprobe_cmd = "modprobe --showconfig | grep blacklist | grep #{@module}"
+      end
+      cmd = inspec.command(modprobe_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.delete("\n") : nil
```

Comment:
@adamleff yes I get that. Sorry I was a little stream of consciousness there. I was asking if you thought that I should return `true` and `nil` in these methods - given that is the logic flow - rather than a string of the `stdout` or `nil`. That might be a better implementation. We are asking the function to return an "answer" but we don't care about the data.